src/generic/colrdlgg.cpp

```diff
@@ -148,6 +172,10 @@ bool wxGenericColourDialog::Create(wxWindow *parent, wxColourData *data)
     if (data)
         m_colourData = *data;
 
+#if !USE_PREVIEW_WITH_ALPHA
+    wxASSERT_MSG( !m_colourData.GetShowOpacity(), wxS("Unable to preview ARGB colours") );
```

Comment:
I don't think it warrants an assert. If we can't allow the user to select alpha channel, we don't, it should just be mentioned in the documentation that this functionality might be not available under all platforms/in all build configurations.